pkg/lang/ir/types.go

```diff
@@ -53,6 +53,7 @@ type Graph struct {
 	Exec            []string
 	Copy            []CopyInfo
 	Mount           []MountInfo
+	entrypoint      []string
```

Comment:
It's better to make it public, I am not sure if we use anywhere.
```suggestion
	Entrypoint      []string
```

Reply:
Fixed